lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -219,36 +219,34 @@ def order_tag_attributes(options)
 
       def build_capture_request(money, authorization, options)
         build_order_modify_request(authorization) do |xml|
-          xml.tag! 'capture' do
+          xml.capture do
             time = Time.now
-            xml.tag! 'date', 'dayOfMonth' => time.day, 'month' => time.month, 'year'=> time.year
+            xml.date 'dayOfMonth' => time.day, 'month' => time.month, 'year'=> time.year
             add_amount(xml, money, options)
           end
         end
       end
 
       def build_void_request(authorization, options)
-        build_order_modify_request(authorization) do |xml|
-          xml.tag! 'cancel'
-        end
+        build_order_modify_request(authorization, &:cancel)
```

Comment:
This is a situation where RuboCop was complaining about the syntax; this seemed to make both RuboCop and the tests happy.